components/brave_wallet_ui/common/slices/endpoints/nfts.endpoints.ts

```diff
@@ -242,6 +243,88 @@ export const nftsEndpoints = ({
         }
       },
       providesTags: ['AutoPinEnabled']
+    }),
+    getSimpleHashSpamNfts: query<BraveWallet.BlockchainToken[], void>({
+        queryFn: async (_arg, _api, _extraOptions, baseQuery) => {
+          try {
+            const { data: api } = baseQuery(undefined)
+            const { braveWalletService } = api
+            const { cache } = baseQuery(undefined)
+
+            const chainIds = (await getAllNetworksList(api)).map(
+              (network) => network.chainId
+            )
+
+            const accounts = (await cache.getAllAccounts()).accounts
+            const spamNfts = (
+              await mapLimit(
+                accounts,
+                10,
+                async (account: BraveWallet.AccountInfo) => {
+                  let currentCursor: string | null = null
+                  const accountSpamNfts = []
+
+                  do {
+                    const {
+                      tokens,
+                      cursor
+                    }: {
+                      tokens: BraveWallet.BlockchainToken[]
+                      cursor: string | null
+                    } = await braveWalletService.getSimpleHashSpamNFTs(
+                      account.address,
+                      chainIds,
+                      account.accountId.coin,
+                      currentCursor
+                    )
+
+                    accountSpamNfts.push(...tokens)
```

Comment:
How long of an array can `tokens` be?

Reply:
We should avoid using the spread operator when pushing due to this: https://github.com/nodejs/node/issues/16870

Reply:
Not more than 1m so this should be fine